src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseAccessEndpoint.java

```diff
@@ -56,6 +63,9 @@ public CaseAccessEndpoint(CaseAccessOperation caseAccessOperation) {
         @RequestParam(value = "userId") final String idSearchingFor
     ) {
         LOG.debug("Finding cases user: {} has access to", idSearchingFor);
+        if (applicationParams.getEnableAttributeBasedAccessControl()) {
```

Comment:
May be it's good to keep this switch inside CaseAccessOperation rather than in the endpoint as CaseAccessOperation has some business logic??

Reply:
Make sense, good one :)  I will implement